gno.land/pkg/gnoweb/components/breadcrumb.gohtml

```diff
@@ -0,0 +1,12 @@
+{{ define "breadcrumb" }}
```

Comment:
Can we consider not committing those generated files and ensure that each install command runs `go generate`?

Reply:
These files are not generated; they are simply Go HTML templates. That aside, we need to push generated files, as they are mostly CSS and JS, and we don't want users to depend on Node to install gnodev or gnoweb. I've added a CI rule to check if generated files are correctly synced.